recipes/spades/3.9.1/meta.yaml

```diff
@@ -4,18 +4,16 @@ package:
 
 source:
   url: http://spades.bioinf.spbau.ru/release3.9.1/SPAdes-3.9.1-Linux.tar.gz  # [linux]
-  md5: 095f4f7dd0e6496156df5d7c9419a70f                                      # [linux]
+  sha256: 92b9d70666989e9ed5f27dbbc03ce90826d7297bd6894e3ab534cc6ca8976d9a   # [linux]
   url: http://spades.bioinf.spbau.ru/release3.9.1/SPAdes-3.9.1-Darwin.tar.gz # [osx]
-  md5: d41d8cd98f00b204e9800998ecf8427e                                      # [osx]
+  sha256: 7471a9dc9c815a57f67df8d3890396b3b8c198c83fe6f2f6fd148386327a60a6   # [osx]
 
 build:
-  number: 0
-  skip: True  # [py36]
+  number: 2
```

Comment:
Since the previous package wasn't actually uploaded, `spades=3.9.1` doesn't exist in the `bioconda` channel, so the build number here should be 0.